subsys/spm/spm_internal.h

```diff
@@ -153,6 +153,26 @@ extern "C" {
 
 #define FLASH_NSC_SIZE_REG(size) FLASH_NSC_SIZE((size) / FLASH_NSC_MIN_SIZE)
 
+#if USE_PARTITION_MANAGER
+#include <pm_config.h>
+#define NON_SECURE_APP_ADDRESS PM_APP_ADDRESS
+#ifdef PM_SRAM_SECURE_SIZE
+#define NON_SECURE_RAM_OFFSET PM_SRAM_SECURE_SIZE
+#else
+#define NON_SECURE_RAM_OFFSET 0
+#endif
+#else
+#include <storage/flash_map.h>
+#define NON_SECURE_APP_ADDRESS FLASH_AREA_OFFSET(image_0_nonsecure)
+/* This reflects the configuration in DTS. */
+#define NON_SECURE_RAM_OFFSET 0x10000
+#endif /* USE_PARTITION_MANAGER */
+
+#define NON_SECURE_FLASH_REGION_INDEX \
+	((NON_SECURE_APP_ADDRESS) / (FLASH_SECURE_ATTRIBUTION_REGION_SIZE))
+#define NON_SECURE_RAM_REGION_INDEX \
+	((NON_SECURE_RAM_OFFSET) / (RAM_SECURE_ATTRIBUTION_REGION_SIZE))
```

Comment:
Curious: Why was this moved? Was anything changed? (it's hard to tell from the diff)

Reply:
Its to get the symbols needed for getting the limits for secure RAM etc, with the PM fix I think we can move it back.

Reply:
Ah, I see the `#include <spm_internal.h>` now :D.